test/plugin/scenarios/dbcp-2.x-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,255 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+segmentItems:
+- serviceName: dbcp-2.x-scenario
+  segmentSize: nq 0
+  segments:
+  - segmentId: not null
+    spans:
+    - operationName: /dbcp-2.x-scenario/case/healthCheck
+      operationId: eq 0
+      parentSpanId: -1
+      spanId: 0
+      spanLayer: Http
+      componentId: 1
+      isError: false
+      spanType: Entry
+      peer: ''
+      skipAnalysis: false
+      tags:
+        - {key: url, value: 'http://localhost:8080/dbcp-2.x-scenario/case/healthCheck'}
+        - {key: http.method, value: HEAD}
```

Comment:
we don't need to validate this segment.

Reply:
> we don't need to validate this segment.

@dmsolr Thank you sir, I have deleted this segment.